lib/checkPullRequestTemplate.js

```diff
@@ -21,6 +21,11 @@ const { pingAndAssignUsers } = require('./utils');
 const KARMA_LINTER = 'linter/karma';
 const EXPLANATION_TEXT = '[Explain here what your PR does and why]';
 const CHECKLIST_TEXT = '## Essential Checklist';
+const PROOF_TEXT = '## Proof that changes are correct';
+const UPLOADED_MEDIA = 'http';
+const PROOF_UNAVAILABLE_TEXT = 'No proof of changes needed because';
```

Comment:
Should this be a regex and maybe be a bit less strict. So allow all cases and bigger whitespaces.

Reply:
Fixed.
The following cases are now covered:
![image](https://user-images.githubusercontent.com/30050862/108677529-d301c580-750f-11eb-8ccf-e1512699725f.png)

Reply:
it is case-insensitive and allows any number of white spaces in between